CarGarage/Program.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+
+public class Program
+{
+    public static void Main()
+    {
+        Car greenCar = new Car("green");
+        Car blueCar = new Car("blue");
+        Car redCar = new Car("red");
+        Garage smallGarage = new Garage(3);
+
+
+        smallGarage.ParkCar(blueCar, 0);
+        smallGarage.ParkCar(greenCar, 1);
+        smallGarage.ParkCar(redCar, 2);
+        Console.WriteLine(smallGarage.Cars);
+        Person p = new Person("James");
+
+        Person p1 = new Person("John");
+        greenCar.enterCar(p1);
+        redCar.enterCar(p);
+        Console.WriteLine("A Passenger {0} rides in a green car.", p1);
+        Console.WriteLine("A Passenger {0} rides in a red car.", p);
+
+    }
+}
+
+class Car
+{
+    public string Color { get; private set; }
+    public Person[] passenger;
+    public Car(string initialColor)
+    {
+        Color = initialColor;
+        this.passenger = new Person[4];
+    }
+    public void enterCar(Person p)
+    {
+        passenger[0] = p;
```

Comment:
This will always put the person in the first position? but will not allow multiple people to enter the car.